src/lib/viewers/controls/media/SettingsControls.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import IconGear24 from '../icons/IconGear24';
+import MediaToggle from './MediaToggle';
+import './SettingsControls.scss';
+
+export default function SettingsControls(): JSX.Element {
+    const [isVisible, setVisible] = React.useState(false);
+
+    const handleClick = (): void => {
+        setVisible(!isVisible);
+    };
+
+    return (
+        <div className="bp-SettingsControls">
+            <MediaToggle className="bp-SettingsControls-toggle" onClick={handleClick} title={__('media_settings')}>
```

Comment:
Do we need to add the little gear rotate animation when toggled?

Reply:
Eventually, but this is mainly a placeholder for now.